app/views/devise/passwords/edit.html.slim

```diff
@@ -11,6 +11,9 @@ h1.h3.my0 = t('headings.passwords.change')
   = f.full_error :reset_password_token
   = f.input :password, label: t('forms.passwords.edit.labels.password'), required: true
   = render 'devise/shared/password_strength'
-  = f.button :submit, t('forms.passwords.edit.buttons.submit'), class: 'mb1'
+  = f.button :submit, t('forms.passwords.edit.buttons.submit'), class: 'mb3'
+
+= accordion('password-info', t('instructions.password.help_text_header')) do
```

Comment:
Sorry just realized which lines you were talking about, I can throw the ones associated with the accodion in a partial if you think that's cleaner